### PR TITLE
Added the scaffold to create task

### PR DESCRIPTION
**Pull request by samuelimoisili**
# Description

Added schema validation to endpoints and a scaffold for the task creation endpoint.

Closes medic/interoperability#14

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by samuelimoisili** [comment]
I want to add a logger and draft out a proper schema for a `patient` and a `task`. I don't know if I should do it in this PR or break it out into other PRs. I also notice last week that OpenHIM isn't forwarding all the request, so I've been trying to debug it.

File: mediator/src/controllers/task.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const axios = require('axios');
```
[the remark this answers is not in the document]
Comment (samuelimoisili) [reply mediator/src/controllers/task.js:1]: Should i rename the endpoint to `service-request`? Or something similar?

File: mediator/src/controllers/patient.js [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-      username: 'interop-client',
-      password: 'interop-password'
+    const res = await axios.post(`${fhirUrl}/Patient`, FHITPatientResource, {auth: {
+      username: fhirUsername,
+      password: fhirPassword,
     }});
-    return {status: res.status, patient: res.data}
+    return {status: res.status, patient: res.data};
   } catch (error) {
     console.error(error);
```
Comment (andrablaj) [inline mediator/src/controllers/patient.js:16]: To be replaced with `logger`?

Comment (samuelimoisili) [reply mediator/src/controllers/patient.js]: Yeah, updated

**Pull request merged**